.tekton/uhc-portal-pull-request.yaml

```diff
@@ -29,588 +30,31 @@ spec:
-        - name: kind
-          value: task
-        resolver: bundles
-      when:
-      - input: $(params.skip-checks)
-        operator: in
-        values:
-        - "false"
-      workspaces:
-      - name: workspace
-        workspace: workspace
-    - name: apply-tags
-      params:
-      - name: IMAGE_URL
-        value: $(tasks.build-image-index.results.IMAGE_URL)
-      - name: IMAGE_DIGEST
-        value: $(tasks.build-image-index.results.IMAGE_DIGEST)
-      runAfter:
-      - build-image-index
-      taskRef:
-        params:
-        - name: name
-          value: apply-tags
-        - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-apply-tags:0.2@sha256:70881c97a4c51ee1f4d023fa1110e0bdfcfd2f51d9a261fa543c3862b9a4eee9
-        - name: kind
-          value: task
-        resolver: bundles
-    - name: push-dockerfile
-      params:
-      - name: IMAGE
-        value: $(tasks.build-image-index.results.IMAGE_URL)
-      - name: IMAGE_DIGEST
-        value: $(tasks.build-image-index.results.IMAGE_DIGEST)
-      - name: DOCKERFILE
-        value: $(params.dockerfile)
-      - name: CONTEXT
-        value: $(params.path-context)
-      runAfter:
-      - build-image-index
-      taskRef:
-        params:
-        - name: name
-          value: push-dockerfile
-        - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-push-dockerfile:0.1@sha256:5446102f233991bdc73451201adf361766d45c638f3d89f19121ae1c2ba8bf17
-        - name: kind
-          value: task
-        resolver: bundles
-      workspaces:
-      - name: workspace
-        workspace: workspace
-    - name: rpms-signature-scan
-      params:
-      - name: image-url
-        value: $(tasks.build-image-index.results.IMAGE_URL)
-      - name: image-digest
-        value: $(tasks.build-image-index.results.IMAGE_DIGEST)
-      runAfter:
-      - build-image-index
-      taskRef:
-        params:
-        - name: name
-          value: rpms-signature-scan
-        - name: bundle
-          value: quay.io/konflux-ci/konflux-vanguard/task-rpms-signature-scan:0.2@sha256:7d1c087d7d33dd97effb3b4c9f3788e4c3138da2032040d69da6929e9a3aaceb
-        - name: kind
-          value: task
-        resolver: bundles
-      when:
-      - input: $(params.skip-checks)
-        operator: in
-        values:
-        - "false"
-    workspaces:
-    - name: workspace
-    - name: git-auth
-      optional: true
-    - name: netrc
-      optional: true
+  - name: unit-tests-script
+    value: |
+      #!/bin/bash
+      set -ex
+      npm install -g yarn
+      yarn install
+      # cap the linter's heap size, as it tends to hog the RAM,
+      # causing the OS to kill the process due to out-of-memory errors
+      NODE_OPTIONS='--max-old-space-size=4096' yarn lint
+      yarn prettier
+      yarn test-no-coverage --run-in-band --no-cache
```

Comment:
I'm assuming that this will not replace the GitHub workflow/action to run unit tests.  Eventually we are going to want to start looking closer at these numbers.  We may want to flag PRs that cause the test coverage to go below a threshold for example.

Reply:
it will eventually replace both jenkins and hopefully the github-action (tho i lost that battle before....)

i assume by "numbers" you mean coverage perc'.  i just followed existing convention and didn't include coverage in PRs.
as mentioned in the desc', coverage will be included in the 'push' pipeline that builds on 'main'.

ultimately it's a matter of choice - if you think we should also (or only) run coverage in PR PLs, speak now!  :)

Reply:
okay i'll merge for now, and leave the coverage decisions for future work.